src/dom-util.js

```diff
@@ -0,0 +1,238 @@
+  if (shot.ship) {
+    type = 'boom';
+    if (shot.ship.hp === 0 && shot.board === 'opp') {
+      const ship = document.createElement('img');
+      ship.className = `ship ship-${shot.ship.length}`;
+      if (!shot.ship.horizontal) { ship.classList.add('v-ship'); }
+      ship.src = `../src/images/ship${shot.ship.length}.png`;
+      document.querySelector(`.opp[data-x="${shot.ship.head.x}"][data-y="${shot.ship.head.y}"]`).appendChild(ship);
+    }
+  }
+  const effect = document.createElement('img');
+  effect.className = 'effect';
+  effect.src = `../src/images/${type}.png`;
+  const cell = document.querySelector(`.${shot.board}[data-x="${shot.x}"][data-y="${shot.y}"]`);
+  cell.classList.remove('darkened');
+  cell.appendChild(effect);
+}
+
+function disableBoard(boardDiv) {
+  boardDiv.childNodes.forEach((e) => e.style.pointerEvents = 'none');
+}
+
+function enableBoard(boardDiv) {
+  boardDiv.childNodes.forEach((e) => e.style.pointerEvents = 'auto');
+}
+
+function createBoard(game) {
+  const board = document.createElement('div');
+  board.className = 'board';
+  for (let i = 0; i < 10; i++) {
+    for (let j = 0; j < 10; j++) {
+      const cell = document.createElement('div');
+      if (game) {
+        cell.className = 'opp cell darkened';
+        cell.addEventListener('click', () => {
+          markShot(game.makeMove(j, i));
+          if (game.hasWinner()) {
+            Swal.fire({
+              title: 'Game Over',
+              text: 'You win!',
+              type: 'success',
+              confirmButtonColor: '#666',
+            });
+            disableBoard(board);
+          } else if (!game.currentPlayer.human) {
+            disableBoard(board);
+          }
+          if (!game.currentPlayer.human) {
+            while (!game.currentPlayer.human) {
+              markShot(game.randomMove());
+            }
+            if (game.hasWinner()) {
+              Swal.fire({
+                title: 'Game Over',
+                text: 'You lose.',
+                type: 'error',
+                confirmButtonColor: '#666',
+              });
+            } else {
+              enableBoard(board);
+            }
+          }
+        });
+      } else {
+        cell.className = 'own cell dropzone';
+      }
+      cell.setAttribute('data-x', j);
+      cell.setAttribute('data-y', i);
+      board.appendChild(cell);
+    }
+  }
+  return board;
+}
+
+function start(game) {
+  randomizeBoard(game.p2.board);
+  game.start();
+  if (game.started) {
+    document.querySelector('#startBtn').style.pointerEvents = 'none';
+    document.querySelector('#startBtn .btn').style.backgroundImage = "url('../src/images/btn-pressed.png')";
+    enableBoard(document.querySelectorAll('.board')[1]);
+    disableBoard(document.querySelectorAll('.board')[0]);
+    Swal.fire({
+      title: 'Game Started',
+      text: 'Good luck!',
+      type: 'warning',
+      confirmButtonColor: '#666',
+    })
+  }
+}
+
+function reset(game) {
```

Comment:
[Optional] According to function name it doesn't look like that it's a part of domUtils.